portfolio/src/main/webapp/blogging.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="page">
+      <div class="top-navigation">
+        <a href="index.html">Home</a>
+        <a href="leadership.html">Leadership</a>
+        <a href="projects.html">Projects</a>
+        <a class="active" href="blogging.html">Blogging</a>
+        <a href="about.html">About</a>
+        <div class="top-navigation-right">
+          <a href="https://www.linkedin.com/in/gabriel-stewart-311206171">
+            <img src="images/linkedin-logo.png" alt="Linkedin Account">
+          </a>
+          <a href="https://github.com/glstewart17">
+            <img src="images/github-logo.png" alt="Github Account">
+          </a>
+        </div>
+      </div>
+      <div id="content">
+        <h2>Blogging</h2>
+        <p>"Some archeologists believe that Stonehenge - the mysterious arrangement of enormous elongated stones 
+            in England - is actually a crude effort by the Druids to build a computing device." - Dave Barry
+        <p>My blogs are no great feat. But like Stonehenge, maybe they will make you wonder.
+        <div class="column">
+          <a href="https://sites.psu.edu/gabespassionblog/">
+            <div class="card">
+              <img src="images/farm.jpg" alt="Farm Land">
+              <div class="card-container">
+                <h4><b>Amish Tech</b></h4> 
+                <p>During my first semester at Penn State, I took some time to explore the technological happenings 
+                  of Lancaster County. While it is a very rural area, I've learned that there is growing tech presence.
+                  My posts were primarily centered around tech-related events and groups that hosted gatherings  
+                  for the public. Additionally, I chose to highlight several promising tech companies in Lancaster.
+              </div>
+            </div>
+          </a>
+        </div>
+        <div class="column">
+          <a href="https://sites.psu.edu/glspassionblog/">
```

Comment:
SUCH a cool blog!! I myself recently read about General vs Narrow AI and was super intrigued. Curious to hear what else on the topic, let's discuss AI more at our next 1:1.

Also, the AI we're doing on AutoFi is very similar to that required for self-driving cars: sign detection, lane marker detection, etc. I can tell you more about this and point you to presentations if you're interested.

Reply:
That would be great! I know I would be interested in learning more about AutoFi.